docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -22,15 +22,16 @@ have to configure `.kube/config` properly.
+| dockerImageTag                  | string                                                  | current nussknacker version        |                                                                                                                                                                                                                                                                     |
+| scalingConfig                   | {fixedReplicasCount: int} or {tasksPerReplica: int}     | { tasksPerReplica: 4 }             | see [below](#configuring-replicas-count)                                                                                                                                                                                                                            |
+| configExecutionOverrides        | config                                                  | {}                                 | see [below](#overriding-configuration-passed-to-runtime)                                                                                                                                                                                                            |
+| k8sDeploymentConfig             | config                                                  | {}                                 | see [below](#customizing-k8s-deployment)                                                                                                                                                                                                                            |
+| nussknackerInstanceName         | string                                                  | {?NUSSKNACKER_INSTANCE_NAME}       | see [below](#nussknacker-instance-name)                                                                                                                                                                                                                             |
+| logbackConfigPath               | string                                                  | {}                                 | see [below](#configuring-runtime-logging)                                                                                                                                                                                                                           |
+| nameOfCommonConfigMapForLogback | string                                                  | {}                                 | see [below](#configuring-runtime-logging)                                                                                                                                                                                                                           |
```

Comment:
maybe just commonConfigMapForLogback ? but if you prefer this one just resolve and merge

Reply:
I don't like this name either. changed